### PR TITLE
CB - Creating the ground for future decompress bug fix

### PR DESCRIPTION
**Pull request by roysagi**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/19544 - does not fix it!!!

## Description
The fix for the bug is on server side (https://github.com/demisto/server/pull/13807).
This is the content update that should come out along side it.

## Does it break backward compatibility?
   - No

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/carbon_black_unzip/Integrations/integration-Carbon_Black_Enterprise_Response.yml)

### CODE REVIEW

**Comment by roysagi** [comment]
@David-BMS - FYI
Again this does not fix the bug but does add a temporary description to the 'decompress' argument. I think this argument should be set to a default 'yes' - but I leave it up to you to decide.

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml [hunk 1]
```diff
@@ -1364,8 +1401,8 @@ script:
       predefined:
       - "yes"
       - "no"
-      defaultValue: "yes"
       description: Whether to decompress results
```
Comment (ronykoz) [inline Integrations/integration-Carbon_Black_Enterprise_Response.yml]: add deprecated to the command in general

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml [hunk 2]
```diff
@@ -1417,7 +1454,7 @@ script:
       description: Description of the signature.
       type: string
     description: Retrieve a binary from CarbonBlack based on hash. Returned file will
-      be places in the war room.
+      be places in the war room. Deprecated - please use cb-binary-download instead.
```
Comment (ronykoz) [inline Integrations/integration-Carbon_Black_Enterprise_Response.yml]: we should first mention the deprecated section, and only then the explanation itself

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml [hunk 3]
```diff
@@ -1900,5 +1937,58 @@ script:
     - contextPath: CbResponse.Watchlists.CbWatchlistID
       description: Watchlist ID
     description: Retrieve watchlist in Carbon black Response.
+  - name: cb-binary-download
+    arguments:
+    - name: md5
+      required: true
+      default: true
+      description: MD5 File Hash
+    - name: summary
```
Comment (ronykoz) [inline Integrations/integration-Carbon_Black_Enterprise_Response.yml]: should be with predefined options of yes and no

File: Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+ -
```
Comment (ronykoz) [inline Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md]: Added new command `cb-binary-download` which replaces `cb-binary-get`

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml (hunk 1 above)
Comment (ronykoz) [reply Integrations/integration-Carbon_Black_Enterprise_Response.yml]: ?

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml (hunk 2 above)
Comment (roysagi) [reply Integrations/integration-Carbon_Black_Enterprise_Response.yml]: done

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml (hunk 3 above)
Comment (roysagi) [reply Integrations/integration-Carbon_Black_Enterprise_Response.yml]: done

File: Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md (hunk 4 above)
Comment (roysagi) [reply Integrations/integration-Carbon_Black_Enterprise_Response_CHANGELOG.md]: done

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml (hunk 1 above)
Comment (roysagi) [reply Integrations/integration-Carbon_Black_Enterprise_Response.yml]: done

File: Integrations/integration-Carbon_Black_Enterprise_Response.yml [hunk 5]
```diff
@@ -1416,8 +1453,9 @@ script:
     - contextPath: File.Signature.Description
       description: Description of the signature.
       type: string
-    description: Retrieve a binary from CarbonBlack based on hash. Returned file will
-      be places in the war room.
+    description: Deprecated - please use cb-binary-download instead. Retrieve a binary
+      from CarbonBlack based on hash. Returned file will be places in the war room.
```
Comment (kirbles19) [inline Integrations/integration-Carbon_Black_Enterprise_Response.yml]: Deprecated. Use the cb-binary-download command instead.

**Comment by kirbles19** [comment]
@roysagi Done.

**Pull request merged**